irida_staramr_results/amr_downloader.py

```diff
@@ -24,6 +26,12 @@ def download_all_results(irida_api, project_id, output_file_name, mode_append):
     if len(amr_completed_analysis_submissions) < 1:
         logging.warning(f"No completed amr analysis submission type for project id [{project_id}].")
 
```

Comment:
I'm thinking of having an `else` statement here because we want to end the program if there is nothing to download. Is it good practice to have most of the core code (which all the following code after this line) in an `else` statement?

An alternative idea I thought about is having the conditional to throw a flag and have that handled by the `cli.py`, where we `sys.exit(1)`.

Reply:
Would it work to simply add a `return` statement?

Reply:
Yup that would work.